<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>300</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>500</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWebView" name="webView">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnAnt">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>204</width>
              <height>50</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
            <property name="autoRepeatInterval">
             <number>100</number>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSig">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>204</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>btnSig</zorder>
         <zorder>btnAnt</zorder>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QWidget" name="verticalWidget_2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>500</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>27</number>
        </property>
        <property name="topMargin">
         <number>38</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>350</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>560</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Ocultar Men√∫</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
